code/modules/atmospherics/machinery/air_alarm/air_alarm_interact.dm

```diff
@@ -71,14 +71,16 @@
 	if(machine_stat & (NOPOWER|BROKEN))
 		to_chat(user, span_warning("It does nothing!"))
 	else
+		if(HAS_SILICON_ACCESS(user))
+			locked = !locked
+			return
```

Comment:
```suggestion
			locked = !locked
			to_chat(user, span_notice("You [ locked ? "lock" : "unlock"] the air alarm interface."))
			return
```

should still give the feedback

Reply:
APCs don't do that though, so it would introduce an inconsistency, and unlike a carbon swiping an ID the UI is guaranteed to already be open to see the button change states (also like an APC).

Reply:
does it auto-open the UI? the code didn't suggest that. if it does then it's whatever

Reply:
Maybe we are talking about different things?
This code does not, but the UI has to be open already to use the button. I believe when an ID swipe unlocks it also opens for QoL. AFAIK only silicons and ghosts can see the button.
This PR is for that button:
![image](https://github.com/tgstation/tgstation/assets/64715958/7a49f8ca-0d05-4b9c-be58-9509123abecf)